homeassistant/components/mqtt/binary_sensor.py

```diff
@@ -157,7 +157,7 @@ async def async_will_remove_from_hass(self) -> None:
         """Remove exprire triggers."""
         # Clean up expire triggers
         if self._expiration_trigger:
-            _LOGGER.debug("Clean up expire after trigger for %s", self.entity_id)
+            _LOGGER.info("Clean up expire after trigger for %s", self.entity_id)
```

Comment:
This is not user facing information.
```suggestion
            _LOGGER.debug("Clean up expire after trigger for %s", self.entity_id)
```